README.md

```diff
@@ -1,11 +1,73 @@
 # README
 
-Add your handler readme here. Remember to include the following:
+## What is this?
 
-- Tell people how to install it (e.g. pip install ...).
-- Be clear about the purpose of the template handler, its capabilities and limitations.
-- Tell people how to use it.
-- Give examples of the template handler in use.
+`sceptre-cdk-handler` is a TemplateHandler for Sceptre (versions 2.7 and up) that lets you use a
+Python AWS CDK template as a stack's template.
 
-If you have any questions or encounter an issue
-[please open an issue](https://github.com/Sceptre/project/issues/new)
+This template handler will use the AWS CDK to synthesize the CDK stack into a CloudFormation template
+and then run `npx cdk-assets` to publish any required assets to S3/ECR.
+
+**By using the CDK Handler, you are letting CDK synthesize a template, and upload artifacts to S3 and ECR
+and then using Sceptre to actually do the deployment of the template to a stack.**
+In other words, by using this handler with Sceptre, _you skip ever using `cdk deploy`; It's not needed_.
+
+By using this handler, you can now use CDK templates with all your favorite Sceptre commands, like
+`launch`, `validate`, `generate`, and `diff` (along with all the rest)!
+
+## How to install sceptre-cdk-handler
+
+Simply `pip install sceptre-cdk-handler`. **Additionally, you need the [AWS CDK](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html) installed along with its prerequisites and the [CDK Assets](https://www.npmjs.com/package/cdk-assets) NPM packacge**.
+
+## How to use sceptre-cdk-handler
+
+The template "type" for this handler is `cdk`. 
+
+### Arguments:
+
+* `path` (string, required): The path to the CDK template.
+* `context` (object, optional): The context for the CDK Stack. See [CDK Context](https://docs.aws.amazon.com/cdk/v2/guide/context.html) for further info.
+
+#### CDK Template Requirements
+
+1. The CDK template must only contain one CDK Stack class, and the class must be called `CdkStack`. 
```

Comment:
Just a side-note: You could make the "class-name" a template handler parameter and make it default to `CdkStack`. Here's why I think that would be useful:

Sometimes in CDK, people design multiple stacks, with one stack referencing the resources on another stack. In those cases, it might be the case that someone has a single Python file with multiple CDK stacks defined with different names. CDK will use the fact that one stack's resources are being referenced in another stack and change the templates accordingly.

Within the context of Sceptre, then, one might want to create a stack for "StackA" and a stack for "StackB" and then add an explicit dependency between those stacks so we deploy them in the right order. It would be a lot easier to _default_ to the stack named "CdkStack" but make it possible to reference "StackA" or "StackB" in the python file if necessary.

Reply:
OK, `class_name` property added.